website/static/js/osfHelpers.js

```diff
@@ -746,7 +746,7 @@ var confirmDangerousAction = function (options) {
                 className: 'btn-default'
             },
             success: {
-                label: 'Confirm',
+                label: options.confirmBtnLabel,
```

Comment:
This makes the options inconsistent. `confirmDangerousAction` takes the same options as `bootbox.dialog`; why make `confirmBtnLabel` a special case?

Instead, you can override `buttons` in `ProjectSettings.getConfirmationCode`, just as you would with bootbox.